### ISSUE TITLE
Request: autocorrect/text prediction

### ISSUE THREAD

**Issue by GXboy12345** [Igalia/wolvic#737]
I know it's a big ask, but an autocorrect/text prediction section on the keyboard would be SUPER useful. For example, right now! I'm using my headset to write this feature request, and it's really painful to just peck, peck, peck one by one and correct every little mistake. I understand if this is low-priority or if you can't/don't want to implement it, but I think that it's a feature a lot of us would like to have. Please consider this :)

**Comment by svillar** [comment]
Yeah, I agree with you that typing in VR is not precisely a pleasant experience. And with hand tracking is much more painful. Some other users also asked for the voice recognition to be available also for general text input and not only search. We'll take your suggestions into consideration indeed.

**Comment by javifernandez** [comment]
This may be duplicated of #660

**Comment by GXboy12345** [comment]
I'm not particularly a fan of voice dictation... There are a few autocorrect/suggestion APIs/Libraries that are free to use...

**Comment by GXboy12345** [comment]
Could this be reopened?

**Comment by svillar** [comment]
Sure it isn't really a dup of #660, let's keep it open although there are no current plans to work on that. Happy to accept contributions!
